scripts/release_scripts/repo_specific_changes_fetcher_test.py

```diff
@@ -102,16 +106,22 @@ def mock_run_cmd(unused_cmd: str) -> str:
     def test_get_changes(self) -> None:
         def mock_get_changed_schema_version_constant_names(
             unused_release_tag_to_diff_against: str
-        ) -> List[str]:
-            return ['version_change']
+        ) -> Optional[List[str]]:
+            if self.call_counter:
+                return ['version_change']
+            return None
         def mock_get_setup_scripts_changes_status(
             unused_release_tag_to_diff_against: str
-        ) -> Dict[str, bool]:
-            return {'setup_changes': True}
+        ) -> Dict[str, bool] | None:
```

Comment:
Can you use `Optional` here and below too?

Reply:
Done